### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alyssa-toay**
Updating documentation for using SSH for remote execution using jupyter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,39 @@ Notebooks uses AI Platform version 1.14, we had to code in Python 3.5 (no
 f-strings or type hinting)
 
 ### Jupyter via remote execution into AI Platform Notebook Instance
-Coming Soon - Some documentation already complete by team
+Prerequisites: 
+1. You must have Anaconda/Jupyter Notebook installed
```
Comment (ReneeErnst) [inline README.md]: I think it would be clearer to say that you must have python (I think 3.5+) with jupyter

File: README.md [hunk 2]
```diff
@@ -27,7 +27,39 @@ Notebooks uses AI Platform version 1.14, we had to code in Python 3.5 (no
 f-strings or type hinting)
 
 ### Jupyter via remote execution into AI Platform Notebook Instance
-Coming Soon - Some documentation already complete by team
+Prerequisites: 
+1. You must have Anaconda/Jupyter Notebook installed 
+2. If using a windows machine, you must have PuTTY installed
```
Comment (ReneeErnst) [inline README.md]: Is there anything different if using a mac? I know we only have windows here, but if we have info to make this more useful to others that stumble on the repo, that would be great!

File: README.md [hunk 3]
```diff
@@ -27,7 +27,39 @@ Notebooks uses AI Platform version 1.14, we had to code in Python 3.5 (no
...
+
+Helpful Links: 
+https://cloud.google.com/ai-platform/notebooks/docs/ssh-access
+
+Instructions:
+1. On the AI Platform Notebook Instance you created,
+open your VM Instance details.
+On the dropdown for Remote access, select "view gcloud command"
+Example: 
+gcloud compute --project "project-name" ssh --zone "us-east1-d" "instance-name"
```
Comment (ReneeErnst) [inline README.md]: I think putting this a code bloc would be helpful. For example:
```
gcloud compute --project <project-name> ssh --zone <zone> 
```
Does the command require the quotes? If not, I personally find using the <> to be clearer that this section is where you put your own inputs and quotes are not needed (I think this is also consistent with google documentation).

Comment (ReneeErnst) [inline README.md]: One more question on this line - is specifying a zone required for instances that are already created with a zone? I might be forgetting something here, but curious.

**Review by ReneeErnst — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alyssa-toay) [reply README.md]: Yes- mac's have a built in ssh client so PuTTY is not needed - I will update!

File: README.md (hunk 3 above)
Comment (alyssa-toay) [reply README.md]: Good call out! Yes, the command requires the quotes and the zone. I'll call that out.

File: README.md (hunk 2 above)
Comment (ReneeErnst) [reply README.md]: Oh yeah! It's been so long since I've used a Mac!

File: README.md (hunk 3 above)
Comment (ReneeErnst) [reply README.md]: Interesting that the quotes are needed given that for the gcloud ai platform jobs command they aren't! There might be something I'm unaware of there. I'm also curious about the required zone when the instance is already created. I wonder if you can change the zone for an existing notebook this way, or if the instance was created in one zone and you specify a different one here if you would get an error.

**Review by ReneeErnst — APPROVED** [review]

**Pull request merged**